src/lib/schematics/shell/index.ts

```diff
@@ -78,7 +78,7 @@ function addBodyMarginToStyles(options: Schema) {
     const buffer = host.read(stylesPath);
     if (buffer) {
       const src = buffer.toString();
-      const insertion = new InsertChange(stylesPath, src.length, `\nbody { margin: 0; }\n`);
+      const insertion = new InsertChange(stylesPath, src.length, `\nbody { margin: 0; font-family: 'Roboto', sans-serif; }\n`);
```

Comment:
I'm not sure this should happen by default. The install schematic currently makes things available without _changing_ the existing app, and I think it's best to preserve that. We can add something like a `--typography` option, though.

@crisbeto would setting `.mat-body` be the more appropriate thing to do here?

Reply:
We don't have a `mat-body`, but setting `mat-typography` would propagate all the typography styles.

Reply:
@jelbourn - You told me to add this in the original issue.

> We should add a style to make the font Roboto...

Reply:
Ha, I'm apparently fickle. Today I'm on board with it being there by default, but think we should do it with a css class rather than adding a new style on the body directly. I'm fine with that being in a follow-up PR, though.